server/web/config.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+class Config():
+    SECRET_KEY = '%yt1uvp5-v52s=+(kt)fan$nne$b4s5u(+*o)b(0cd_)hkzp30';
```

Comment:
How secret is this secret key?

Reply:
Its our base config. We will overwrite reading from an envar in production